Poll.py

```diff
@@ -0,0 +1,48 @@
+import discord
+from settings import *
+from checks import *
+
+
+class Poll(commands.Cog):
+	"""TODO: A suite of commands providing users with tools to more easilly get the community's opinion on an idea"""
+	def __init__(self, bot):
+		self.bot = bot
+		
+	@commands.Cog.listener()
+	async def on_reaction_add(self, reaction, user):
```

Comment:
switch to `on_raw_reaction_add` ?

Reply:
Completely forgot to do this.

Reply:
This will be added once #9 will be merged